src/ngx_http_lua_api.c

```diff
@@ -11,6 +11,18 @@
 #include "ngx_http_lua_util.h"
 
 
+typedef struct ngx_http_lua_shm_zone_ctx_s {
+    ngx_log_t                *log;
+    ngx_http_lua_main_conf_t *lmcf;
+
+    u_char                    data; /* ngx_shm_zone_t */
+} ngx_http_lua_shm_zone_ctx_t;
+
+
+static ngx_int_t
```

Comment:
Style: we don't need a line break here for C function declarations (we only need the break after the return value type for C function definitions).